### ISSUE TITLE
Support reconnect request from gremlin client

### ISSUE THREAD

**Issue by siyuan0322** [alibaba/GraphScope#3289]
When gremlin client thinks a remote is unavailable, it will try to reconnect by send a ping message.
But we currently doesn't handle the ping message so that it will consider the host unavailable even if the hosts received the ping message.

The ping message should be handled correctly, i.e. the client should be able to reconnect after the hosts become available.

**Comment by shirly121** [comment]
The Gremlin Client allows users to specify queries for validation. The specific interface is `validationRequest(string)` in `Cluster`. The Java client configuration example is as follows:
```
Cluster cluster = Cluster.build().validationRequest("g.V().limit(1)").create();
Client client = cluster.connect();
client.submit("g.V()");
...
```

**Comment by siyuan0322** [comment]
Fixed by #3315
